packages/devtools_app/lib/src/theme.dart

```diff
@@ -122,7 +122,7 @@ bool isValidLightColor(Color color) {
 const defaultButtonHeight = 36.0;
 const buttonMinWidth = 36.0;
 
-const defaultIconSize = 16.0;
+const defaultIconSize = 18.0;
```

Comment:
is this intentional? Have you validated all the UI still looks good with this larger icon size?